AQA_Lab_Task/DoctorAppointment/StringFormatting.cs

```diff
@@ -0,0 +1,20 @@
+using System.Diagnostics;
+
+namespace DoctorAppointment;
+
+public static class StringFormatting
+{
+    public static string? Formatting(string? input)
```

Comment:
Looks like method just transform 1st letter to uppercase, so give convenient name for this method